src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -254,6 +254,17 @@ the app's preview in task switchers. The `theme_color` should match the
   {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/8mkBdT3O0FZLo0PUppvv.png", alt="An example of a PWA window with custom theme_color.", width="800", height="196", class="w-screenshot" %}
 </figure>
 
+In Chrome 93, you can adjust this color based on a media query with the `media`
```

Comment:
```suggestion
In Chrome&nbsp;93, you can adjust this color based on a media query with the `media`
```

Reply:
This is something Tom has been doing for a long time.

Reply:
Fixed.